2025-07-11 16:24:07,700 INFO App started on linux with Python 3.13.3 (main, Jun 16 2025, 18:15:32) [GCC 14.2.0]
2025-07-11 16:25:33,673 INFO App started on linux with Python 3.13.3 (main, Jun 16 2025, 18:15:32) [GCC 14.2.0]
2025-07-11 16:28:00,485 INFO App started on linux with Python 3.13.3 (main, Jun 16 2025, 18:15:32) [GCC 14.2.0]
2025-07-11 16:29:11,846 WARNING Retry 1/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:14,080 WARNING Retry 2/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:16,329 WARNING Retry 3/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:16,330 ERROR File list error: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:18,369 WARNING Retry 1/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:19,692 WARNING Retry 1/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:20,622 WARNING Retry 2/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:22,055 WARNING Retry 2/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:22,977 WARNING Retry 3/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:22,978 ERROR File list error: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:24,411 WARNING Retry 3/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:24,411 ERROR File list error: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:27,787 WARNING Retry 1/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:30,042 WARNING Retry 2/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:32,295 WARNING Retry 3/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:29:32,296 ERROR Search failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:30:45,102 WARNING Retry 1/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:30:47,126 WARNING Retry 1/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:30:48,766 WARNING Retry 2/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:30:49,404 WARNING Retry 2/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:30:51,144 WARNING Retry 3/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:30:51,145 ERROR Search failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:30:51,758 WARNING Retry 3/3 failed: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:30:51,759 ERROR File list error: An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: The specified bucket does not exist
2025-07-11 16:36:43,906 INFO App started on linux with Python 3.13.3 (main, Jun 16 2025, 18:15:32) [GCC 14.2.0]
2025-07-11 17:01:46,201 INFO App started on linux with Python 3.13.3 (main, Jun 16 2025, 18:15:32) [GCC 14.2.0]
2025-07-11 17:03:07,145 ERROR Offline mode detected.
2025-07-11 17:03:10,150 ERROR Offline mode detected.
2025-07-11 17:03:13,154 ERROR Offline mode detected.
2025-07-11 17:03:20,577 ERROR Offline mode detected.
2025-07-11 17:03:48,053 ERROR Offline mode detected.
2025-07-11 20:33:25,807 INFO Created folder test_folder in navoahansen/MichaelHansen/test_folder/
